HIP/hip-1.md

```diff
@@ -283,69 +285,71 @@ if email or username are not given.
+
+Note: This supercedes the old field `needs-council-approval`. While HIPs prior to Hiero migration maintain this old field for historical referencing; it is no longer used in new HIP headers. Hiero Governing Council TechComm maintains a separate decision-making process on which HIPs to integrate on the Hedera network.
 
 #### `created` header
 
-The created header records the date that the HIP was assigned a number. This header should be in ISO 8601 format (yyyy-mm-dd), e.g. 2019-09-16.
+The `created` header records the date that the HIP was assigned a number. This header should be in ISO 8601 format (`yyyy-mm-dd`), e.g., `2019-09-16`.
 
 #### `last-call-date-time` header
 
-The anticipated date and time when this HIP will change status to `Accepted`. This header should be in ISO 8601 format expressed in UTC (yyyy-mm-ddThh:mm:ssZ), e.g. 2019-09-16T13:15:30Z. This field is usually set as 14 days from the moment a pull requested is reviewed.
+The anticipated date and time when this HIP will change status to `Last Call`. This header should be in ISO 8601 format expressed in UTC (`yyyy-mm-ddThh:mm:ssZ`), e.g. `2019-09-16T13:15:30Z`. This field is usually set as 14 days from the moment a pull request is reviewed.
 
 #### `updated` header
 
-The updated header records the date(s) when the HIP was updated with "substantial" changes. The header is only valid for HIPs of Draft and Active status. This header should be in ISO 8601 format (yyyy-mm-dd), e.g. 2019-09-16.
+The updated header records the date(s) when the HIP was updated with "substantial" changes. It is only valid for HIPs in `Draft` and `Active` status. This header should be in ISO 8601 format (`yyyy-mm-dd`), e.g., `2019-09-16`.
 
 #### `requires` header
 
-HIPs may have a requires header, indicating the HIP numbers that this HIP depends on.
+HIPs may have a `requires` header, indicating the HIP numbers that this HIP depends on.
 
 #### `superseded-by` and `replaces` headers
 
-HIPs may also have a superseded-by header indicating that a HIP has been rendered obsolete by a later document; the value is the number of the HIP that replaces the current document. The current document must change status to "Replaced" once the superseding HIP is changed to "Final" status. The newer HIP must have a replaces header containing the number of the HIP that it rendered obsolete.
+HIPs may also have a `superseded-by` header indicating that a HIP has been rendered obsolete by a later document; the value is the number of the HIP that replaces the current document. The current document must change status to `Replaced` once the superseding HIP is changed to `Final` status. The newer HIP must have a `replaces` header containing the number of the HIP that it rendered obsolete.
 
 #### `release` header
 
-The release header indicates when the HIP was implemented on the network (vX.Y.Z), e.g. 0.29.0.
+The `release` header indicates when the HIP was implemented in the codebase (vX.Y.Z), e.g., `v0.29.0`.
```

Comment:
Maybe it makes sense to mention that Hiero release numbers are used here and not Hedera release numbers.
In future Hedera mainnet version A will be build based on Hiero version B.